encryption/round5/round5-kem/bench/avx2_r5n1-1kem-0d_opt.yaml

```diff
@@ -0,0 +1,7 @@
+sources:
```

Comment:
@TomerHawk do you think we can conclude from subsection 2.9.9 that the same platform was used to run the avx2 benchmarks like for the optimized implementation? Hardware would support it, I don't see why they would change it.

Reply:
You are right : "The performance was measured the same  way  as  the  non-AVX2  optimized  implementation "